spock/plugins/helpers/inventory.py

```diff
@@ -144,6 +145,16 @@ def __init__(self, ploader, settings):
         # stores the last click action for confirmation
         self.last_click = None
 
+        # to know when the inventory got synced with server
+        self.is_synchronized = False
```

Comment:
It should be an instance property, so it should be undefined when no instance (and thus no inventory to be synchronized) exists.

By the way, other plugins cannot access this anyways, as it is in `InventoryPlugin`, not in `InventoryBase`.